packages/components/src/components/install-guide/install-instructions/Python.vue

```diff
@@ -1,18 +1,37 @@
 <template>
   <div>
     <section>
-      <h4>Installation via <code class="inline">pip</code></h4>
-      <v-code-snippet clipboard-text="pip install appmap" />
-    </section>
+      Manual installation is a good option if:
 
+      <ol>
+        <li>
+          Your project doesn't use <i>requirements.txt</i>, <i>Pipfile</i>, or <i>poetry.lock</i>.
+        </li>
+        <li>You want to install <tt>appmap</tt> from outside of the code editor.</li>
+      </ol>
+    </section>
     <section>
-      <h4>Installation via <code class="inline">poetry</code></h4>
-      <v-code-snippet clipboard-text="poetry add --dev appmap" />
+      <h3>Add the <tt>appmap</tt> package to your project</h3>
+      <div>
+        <v-code-snippet clipboard-text="pip install appmap" />
+      </div>
+      <div>
+        <div class="center-block">or</div>
+        <v-code-snippet clipboard-text="pipenv install --dev appmap" />
+      </div>
+      <div>
+        <div class="center-block">or</div>
+        <v-code-snippet clipboard-text="poetry add --dev appmap" />
+      </div>
     </section>
 
     <section>
-      <h4>Installation via <code class="inline">pipenv</code></h4>
-      <v-code-snippet clipboard-text="pipenv install --dev appmap" />
+      <h3>Generate configuration file <i>appmap.yml</i></h3>
```

Comment:
This section is a proposal, the Python program below doesn't actually exist yet.